src/backend/distributed/metadata/metadata_utility.c

```diff
@@ -452,7 +453,8 @@ DistributedTableSize(Oid relationId, char *sizeQuery, bool failOnError, uint64 *
  * shard placement.
  */
 static bool
-DistributedTableSizeOnWorker(WorkerNode *workerNode, Oid relationId, char *sizeQuery,
+DistributedTableSizeOnWorker(WorkerNode *workerNode, Oid relationId, SizeQueryType
+							 sizeQueryType,
```

Comment:
hit; Lets keep type and variable on the same line

```suggestion
DistributedTableSizeOnWorker(WorkerNode *workerNode, Oid relationId,
							 SizeQueryType sizeQueryType,
```